dashboard/config/routes.rb

```diff
@@ -1059,6 +1059,8 @@
 
     get '/get_token', to: 'authenticity_token#get_token'
 
+    post '/api/aichat', to: 'api#aichat'
```

Comment:
Does this route name need to be updated to match what's in the controller (I see `/openai/chat_completion` listed as the comment in the controller)?

Reply:
Actually, I forgot to update the comment from [Emily's POC pr](https://github.com/code-dot-org/code-dot-org/tree/task/edu-bot-poc-2). From [AI API Spec Brainstorm doc](https://docs.google.com/document/d/1Ie0R_eHJnFzRPFOA558JUof-9ND_SFL28A6cE8gvShA/edit) I saw that they recommended `/api/aichat`.

Reply:
I wouldn't index on the proposed name in the AI API spec too much yet. Given that the AI API and its various applications don't exist yet, personally, I think `/openai/chat_completion` or `/ai/chat_completion` feels like it is more descriptive for now. Also, I don't think there's any reason we can't change this for a while?

Reply:
Sounds good - I'll change back to the more descriptive `/openai/chat_completion`. When this PR is out of draft, I'll ask Seth and Wilkie to take a look so they're good with changes. Thanks!